packages/mobile/src/verify/AlternatingText.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react'
+import { Text, View } from 'react-native'
+
+// how often to swap the text
+const SHOW_TIME = 2 * 1000 // ms
+
+interface Props {
+  style: any
+  primaryText: string
+  secondaryText: string
+}
+
+export default function AlternatingText({ style, primaryText, secondaryText }: Props) {
+  const [time, setTimer] = useState(0)
```

Comment:
From what I see, this state could be removed.

Reply:
much better, thanks @jeanregisser